dotcom-rendering/src/lib/branding.ts

```diff
@@ -183,8 +197,8 @@ export const decideCollectionBranding = ({
 		return undefined;
 	}
 
-	// Ensure each of the card's branding has the same sponsor
-	if (!everyCardHasSameSponsor(brandingForCards)) {
+	const hasMultipleBranding = !everyCardHasSameSponsor(brandingForCards);
+	if (kind !== 'paid-content' && hasMultipleBranding) {
```

Comment:
I'm not sure if this `if` condition is necessary, but feels safer given the unknowns about the other options of `kind`